README.md

```diff
@@ -32,8 +32,8 @@ This C# SDK is automatically generated by the [Swagger Codegen](https://github.c
 - SDK version: 2.3.1
```

Comment:
The readme uses a logo inconsistent with all the other readmes. Please swap it for one from the https://github.com/fingerprintjs/home/tree/main/resources

Reply:
Fixed!